modules/serviceregistry/src/main/java/org/opencastproject/serviceregistry/impl/ServiceRegistryJpaImpl.java

```diff
@@ -2844,8 +2857,13 @@ protected List<ServiceRegistration> getServiceRegistrationsByLoad(String jobType
       filteredList.add(service);
     }
 
-    // Sort the list by capacity
-    Collections.sort(filteredList, new LoadComparator(systemLoad));
+    // Sort the list by capacity and distinguish between composer jobs and other jobs
+    if ("org.opencastproject.composer".equals(jobType)) {
+        Collections.sort(filteredList, new LoadComparatorEncoding(systemLoad));
```

Comment:
I'm surprised that checkstyles isn't complaining here.  This line has at least two extra spaces.  We also do K&R bracing (this isn't checked afaik).

Reply:
I will change this.